scripts/helpers.ts

```diff
@@ -113,8 +114,9 @@ export const setupOrRetrieveGateway = async (
         OVM_L1ERC20Gateway = newGateway.OVM_L1ERC20Gateway
         OVM_L2DepositedERC20 = newGateway.OVM_L2DepositedERC20
     } else {
-        OVM_L1ERC20Gateway = new Contract(l1ERC20GatewayAddress, Def__MyL2DepositedERC20.abi, l1Wallet)
-        const l2ERC20GatewayAddress = await OVM_L1ERC20Gateway.l2ERC20Gateway()
+        const OVM_L1ERC20Gateway_Def = getContractDefinition('OVM_L1ERC20Gateway', false);
+        OVM_L1ERC20Gateway = new Contract(l1ERC20GatewayAddress, OVM_L1ERC20Gateway_Def.abi, l1Wallet)
+        const l2ERC20GatewayAddress = await OVM_L1ERC20Gateway.l2DepositedToken()
         OVM_L2DepositedERC20 = new Contract(l2ERC20GatewayAddress, Def__MyL2DepositedERC20.abi, l2Wallet)
```

Comment:
Any reason why you didn't update the other instance of `Def__MyL2DepositedERC20`?

This shouldn't be an issue if you revert the name of `OVM_L1ERC20Gateway_Def`.